openedx/features/edly/models.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+from django.db import models
+from organizations.models import Organization
+
+ALPHANUMERIC_VALIDATOR = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
+
+
+class EdlyOrganization(models.Model):
+    name = models.CharField(max_length=255)
+    slug = models.CharField(max_length=100, unique=True, validators=[ALPHANUMERIC_VALIDATOR])
```

Comment:
I still cannot see the test for validation error `Only alphanumeric characters are allowed.`.

Reply:
Done